test/source/tests/flaky.ts

```diff
@@ -107,8 +107,11 @@ export const defineFlakyTests = (testVariant: TestVariant, testWithBrowser: Test
       await SettingsPageRecipe.toggleScreen(settingsPage, 'additional');
       const fingerprint = (await settingsPage.read('.good', true)).split(' ').join('');
       const myKeyFrame = await browser.newPage(t, `chrome/settings/modules/my_key.htm?placement=settings&parentTabId=60%3A0&acctEmail=${acctEmail}&fingerprint=${fingerprint}`);
-      const files = await myKeyFrame.awaitDownloadTriggeredByClicking('@action-download-prv');
-      const key = await KeyUtil.parse(Object.values(files).pop()!.toString());
+      const downloadedFiles = await myKeyFrame.awaitDownloadTriggeredByClicking('@action-download-prv');
+      // Key ID is last 16 characters of the fingerprint
```

Comment:
This is actually named `longid` and it is calculated this way for `OpenPGP` keys only. For S/MIME keys we use another approach that helps identify the key from the message data. Note that it may be too long and I filed this issue #3937 previously